### DISCUSSION TITLE
Tag along!

### DISCUSSION THREAD

**Discussion by lakruzz** [kea-dev/dx-intro#5] [General]
### Instructions on how to join is in the readme of [this repository](https://github.com/kea-dev/dx-intro/)

**Comment by MikaelElkiaer** [comment]
Hey, cool presentation yesterday!
I am a process/tool nut - like it seems that you are - so I appreciate all this effort in making the tools work for you.

I had a small note on the use of `gh` for API calls not directly supported yet.

For calls like:

```bash
curl -X POST \
              -H "Accept: application/vnd.github.v3+json" \
              -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
              https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/cancel
```

You can do:

```bash
gh api -X POST /repos/{owner}/{repo}/actions/runs/${{ github.run_id }}/cancel
```

It will handle the authorization, media types, and even pagination for you.

Using the placeholders, such as `{owner}`, allows you to do aliases that work in different repository contexts.
Personally, I often use these aliases to list available packages:

```
gh alias set packages-versions "api --paginate /user/packages/container/{repo}/versions?state=active"
gh alias set packages-versions-org "api --paginate /orgs/{owner}/packages/container/{repo}/versions?state=active"
```

Lots more cool stuff if you see `gh api --help`.

**Reply by lakruzz** [reply]
Thanks 🙏 @MikaelElkiaer 

Really nice feature I overlooked that `gh` continues to amaze me!

I'll add that as an update!
